ansible/configs/ansible-multi-node/sample_vars/hands_on_aap2.yml

```diff
@@ -0,0 +1,164 @@
+---
+#  Ansible Automation Platform 2 Foundations vars
+
+env_type: ansible-multi-node
+cloud_provider: osp                         # Cloud Provider to use
+
+hands_on_aap2: true # Runs the setup-hands-on-aap2.yml during software.yml
+
+student_name: devops # user_info will use this for devops user
+install_student_user: false
+
+agnosticd_inventory_exporter_enable: false # Dump inventory in output_dir
+
+software_to_deploy: none 
+automationcontroller_install: false # Leave thsi task for student    
+install_vscode_server: false # Use container version?
+
+inventory_groups:
+
+  - load_balancers
+  - app_servers
+  - database_servers
+
+common_packages:
```

Comment:
@tonykay common_packages variable is not applicable for RHEL8 images. You can use common_packages_el8 instead.

Reply:
Ah, thanks! Copied that over. Will tweak.